### DISCUSSION TITLE
앉아 있다 이동 시 즉시 이동 가능하게 해주세요

### DISCUSSION THREAD

**Discussion by neopandora** [metamong-friends/metamong-friends.github.io#41] [제안하기]
의자에 앉아있다 이동 버튼을 누르면 자동으로 일어나서 이동하게 하면 편할 것 같습니다.
X를 눌러서 일어선 후에 이동하려고 하니 조금 불편하긴 합니다.

**Comment by ACT-TJ** [comment]
안녕하세요!!

숄더뷰로 가보시면, 앉아있을 경우 키보드를 통해 유저가 보는 카메라를 이동시켜 내 위치에 따른 사각지대를 줄이고, 보고자 하는 각도로 세팅 할 수 있도록 되어 있습니다. 뷰마다 동작 일관성의 이슈도 있고,  앉는 동작을 해제하는 것도 명시적으로 트리거를 주는 쪽이 좀 더 향후 사용에 나을 것이라 판단을 하였습니다.
사용성 개선의견으로 관리하면서 유저분들의 피드백을 좀 더 받아보고 저희도 고민해 보도록 하겠습니다.

의견 너무 감사합니다!!

**Comment by shlee303** [comment]
@neopandora 해당 기능 반영 됐습니다! 좋은 제안 감사합니다~!!!
